sig-auth/annual-report-2021.md

```diff
@@ -0,0 +1,114 @@
+
+- Need to review subproject owners to confirm.
+
+**How do you measure membership? By mailing list members, OWNERs, or something else?**
+
+- We do not currently measure membership. We could start with Slack, mailing list members, and meeting attendance. As of this report, there are 2,136 members of the main SIG Auth Slack channel and 447 members of the mailing list. 
+
+**How does the group measure reviewer and approver bandwidth? Do you need help in any area now? What are you doing about it?**
+
+- We do not currently measure this. Need to look at how other SIGs do this today.
+
+**Is there a healthy onboarding and growth path for contributors in your SIG? What are some activities that the group does to encourage this? What programs are you participating in to grow contributors throughout the contributor ladder?**
+
+- Currently there is no onboarding or growth path. This is something we are working on and learning from other SIGs. We will start by creating a CONTRIBUTING.md file.
+
+**What programs do you participate in for new contributors?**
+
+- We have intro sessions at KubeCon to help new contributors get started.
+
+- We now have weekly issues and PRs triage meetings. New contributors can join to help with new issues and PRs. 
+
+**Does the group have contributors from multiple companies/affiliations? Can end users/companies contribute in some way that they currently are not?**
+
+- Yes. Our chairs, leads, contributors, participants, and subproject owners are from various companies.
+
+## Current initiatives and project health
+
+- Please include links to KEPs and other supporting information that will be beneficial to multiple types of community members. 
+
+**What are initiatives that should be highlighted, lauded, shout out, that your group is proud of? Currently underway? What are some of the longer tail projects that your group is working on?**
+
+- [CSR v1](https://github.com/kubernetes/enhancements/tree/master/keps/sig-auth/1513-certificate-signing-request)
+- [Token Request / bound SA token admission](https://github.com/kubernetes/enhancements/tree/master/keps/sig-auth/1205-bound-service-account-tokens)
+- [client-go auth plugins](https://github.com/kubernetes/enhancements/blob/master/keps/sig-auth/541-external-credential-providers)
+- [external kubelet credential providers](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/2133-kubelet-credential-providers)
+- [New features in Secrets Store CSI driver](https://secrets-store-csi-driver.sigs.k8s.io/introduction.html#features)
+- [Pod Security Policy Replacement](https://github.com/kubernetes/enhancements/issues/2579)
+
+**Year to date KEP work review: What’s now stable? Beta? Alpha? Road to alpha?**
+
+- Stable
+    - BoundServiceAccountToken (v1.22)
+    - Certificates API (v1.19)
+    - TokenRequest (v1.20)
+    - TokenRequestProjection (v1.20)
+    - RootCAConfigMap (v1.21)
+    - ServiceAccountIssuerDiscovery (v1.21)
+- Beta
```

Comment:
client-go auth plugins?

Reply:
Did you want to add client-go auth plugins for GA in 1.22?